pages/common/git-fame.md

```diff
@@ -0,0 +1,28 @@
+# git fame
+
+> Pretty-print Git repository contributions.
+> More information: <https://github.com/casperdcl/git-fame>.
+
+- Calculate contributions for the current Git repository:
+
+`git fame`
+
+- Exclude files/directories that match the specified regular expression:
+
+`git fame --excl "{{regular_expression}}"`
```

Comment:
What would a regular_expression be

Reply:
For example `^git-` to exclude all git subcommand pages.

Reply:
Exactly.

Reply:
on a related note I agree that `--excl` is probably more useful for `tldr` to document than `--incl`.

Reply:
> What would a regular_expression be

I'm on tablet atm can't find the relevant issue, so but we decided instead of 'regex' ish things, just use regular expression

Reply:
Correct @Waples and that's what we're doing here.

Reply:
>on a related note I agree that `--excl` is probably more useful for `tldr` to document than `--incl`.

Yeah, I figured that out, because I _was_ going to document `--incl` only 😆